### ISSUE TITLE
Image Resize

### ISSUE THREAD

**Issue by kiarafbickers** [tea3/hexo-tag-instagram#3]
Feature request for resizing the image to 50% , 75% of screen size, ect.

**Comment by tea3** [comment]
@kiarafrobles 
Can you accept to add the css as follow into your theme ?

```
/* How to simply customze width */
.instagram-wrapper{
  margin: 1em auto;
  width: 50%;             /* Please set width that you like. */
}
```

**Comment by kiarafbickers** [comment]
I can accept css.

**Comment by tea3** [comment]
@kiarafrobles 
Thank you. This plugin outputs instagram code wrapped with `<div class="instagram"></div>`. So , to resize it , please add the css.

```
/* (e.g.) Change the width according to screen size */
@media (min-width: 800px) {
  .instagram-wrapper {
    margin: 1em auto;
    width: 50%;
    /* max-width: 658px */       /* To resize with max-width , please use this code. */
  }
}

@media (min-width: 481px) {
  .instagram-wrapper {
    margin: 1em auto;
    width: 75%;
  }
}

@media (min-width: 0px) and (max-width: 480px) {
  .instagram-wrapper {
    margin: 1em auto;
    width: 100%;
  }
}
```

**Comment by kiarafbickers** [comment]
It works great. :]

![screen shot 2017-10-09 at 1 12 53 am](https://user-images.githubusercontent.com/6563016/31329474-23e900a0-ac8f-11e7-81fc-8a42c833549d.png)

**Comment by xu-song** [comment]
Customization has been implemented in #4
